metatron/src/main/java/uk/gov/ida/eidas/metatron/core/dto/EidasConfig.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.ida.eidas.metatron.core.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.security.KeyStore;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.function.Function;
+
+public class EidasConfig {
+    @JsonProperty
+    private List<EidasCountryConfig> countries = new ArrayList<>();
+
+    @JsonProperty
+    private KeyStore truststore;
+
+    public Collection<EidasCountryConfig> getCountries() {
+        return countries;
+    }
+
+    public KeyStore getKeyStore() {
```

Comment:
`getTrustStore` rather than `getKeyStore`? (It's probably not hugely important, but as the member is `truststore` it strikes me that that's what it's being used for, so probably a less generic variable name?

Reply:
I'm pretty sure we change this in the upcoming PR. If not, we will!